db/migrate/20190924171312_add_benchmark_id_to_profile.rb

```diff
@@ -1,5 +1,11 @@
 class AddBenchmarkIdToProfile < ActiveRecord::Migration[5.2]
+  PHONY_BENCHMARK = Xccdf::Benchmark.find_or_create_by(
+    ref_id: 'phony',
+    version: 'v0.0.0'
+  )
+
   def change
-    add_column :profiles, :benchmark_id, :uuid, null: false
+    add_column :profiles, :benchmark_id, :uuid, null: false,
+      default: PHONY_BENCHMARK.id
```

Comment:
It still fails with `PG::NotNullViolation: ERROR:  column "benchmark_id" contains null values`.
I think you have to `Profile.update_all(benchmark_id: PHONY_BENCHMARK.id)` before you even run the change

Reply:
Unfortunately I get `PG::NotNullViolation: ERROR:  column "benchmark_id" contains null values
: ALTER TABLE "profiles" ALTER COLUMN "benchmark_id" TYPE uuid, ALTER COLUMN "benchmark_id" SET DEFAULT '5e791411-38aa-4ff0-9618-48cf355ea5a0', ALTER COLUMN "benchmark_id" SET NOT NULL` - as you can see, the default is a valid benchmark ID